pyciemss/interfaces.py

```diff
@@ -33,6 +33,32 @@
 
 warnings.simplefilter("always", UserWarning)
 
+DEFAULT_ALPHA_QS = [
```

Comment:
Could you import this from `pyciemss.integration_utils.result_processing` instead of redefining it here?